duckduckgo/macos-browser
# Add new tab settings to appearance menu

Task/Issue URL: https://app.asana.com/0/0/1204863486702604/f

**Description**: Adds options to show or hide new tab sections from the settings appearance menu

**Steps to test this PR**:
1. On the new tab click on the button right icon try select and deselect sections and check that it works (try and restart the app to check is persisted) on deselect check pixel is sent
2. Right click in the new tab page  try select and deselect sections and check that it works (try and restart the app to check is persisted) on deselect check pixel is sent
3. Go to the appearance setting, there should be a new tab section check looks like in the task including copy. try select and deselect sections and check that it works (try and restart the app to check is persisted) on deselect check pixel is sent



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
